source/authentication/email-password.txt

```diff
@@ -314,68 +310,63 @@ password reset email:
 Run a Password Reset Function
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you configure the provider to :guilabel:`Run a Password Reset
-Function`, {+service-short+} runs a :doc:`{+service-short+} Function </functions>` that you define
-whenever the client SDK ``callResetPasswordFunction()`` function is
-called. {+service-short+} passes this function unique confirmation tokens and data
-about the user which allows you to define custom logic to reset a user's
-password. The value that you return from the function determines whether
-{+service-short+} immediately resets the user's password or requires additional
+You can configure the provider to :guilabel:`Run a Password Reset
+Function`. You define a :doc:`Function </functions>` for {+service-short+} to run when you ``callResetPasswordFunction()`` in the SDK. 
+{+service-short+} passes this function unique confirmation tokens and data
+about the user. These allow you to define custom logic to reset a user's
+{+service-short+} immediately resets the user's password. You can require additional
```

Comment:
Oops! Some more iffy content here. This renders as this line in the staging build:
> These allow you to define custom logic to reset a user's App Services immediately resets the user's password.

I'd probably say instead something like:
> Use these values to define custom logic to reset a user's password. App Services can immediately reset the user's password. Or you can require additional confirmation from the client application.